pkg/instrumentation/sdk.go

```diff
@@ -419,8 +422,22 @@ func chooseServiceName(pod corev1.Pod, resources map[string]string, index int) s
 	return pod.Spec.Containers[index].Name
 }
 
+func chooseLabelOrAnnotation(pod corev1.Pod, resource attribute.Key, labelKey string) string {
```

Comment:
The parameter name is `labelKey`, but when invoked the `constants.AnnotationAppName` value is used.  Then when indexing the annotations the resource is used, not the `labelKey`.  Is this wired up correctly?

Reply:
labelkey is looked up in labels:

https://github.com/open-telemetry/opentelemetry-operator/blob/bef08935239ce6c7859fa9c0d7ee59f074c8c306/pkg/instrumentation/sdk.go#L429

Reply:
changed constants to have `label` prefix